base/linalg/symmetric.jl

```diff
@@ -337,7 +337,13 @@ det(A::RealHermSymComplexHerm) = real(det(bkfact(A)))
 det(A::Symmetric{<:Real}) = det(bkfact(A))
 det(A::Symmetric) = det(bkfact(A))
 
-\(A::HermOrSym{<:Any,<:StridedMatrix}, B::StridedVecOrMat) = \(bkfact(A), B)
+\(A::HermOrSym{<:Any,<:StridedMatrix}, B::AbstractVector) = \(bkfact(A), B)
```

Comment:
where would abstractvector or abstractmatrix that isn't strided have dispatched to before this?